ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.15.yaml

```diff
@@ -70,6 +70,32 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: azure-ipi-marketplace-image-gen1-vm-type-gen1-f14
```

Comment:
how about change to azure-ipi-marketplace-image-gen1-vm-type-gen1-f28

Reply:
only set f28 on 4.14 job, since we are working on 4.15 full function test, I think it's better to set f14, once it is GA, we can change to f28 then.